pcsx2/VMManager.cpp

```diff
@@ -3016,11 +3016,6 @@ void VMManager::WarnAboutUnsafeSettings()
 	}
 	if (EmuConfig.GS.UpscaleMultiplier < 1.0f)
 		append(ICON_FA_TV, TRANSLATE_SV("VMManager", "Upscale multiplier is below native, this will break rendering."));
-	if (EmuConfig.GS.HWMipmap != HWMipmapLevel::Automatic)
```

Comment:
This probably shouldn't be removed, but instead changed to say "Mipmapping is disabled. This may break rendering in some games."